csrankings-h.csv

```diff
@@ -943,6 +943,7 @@ Huiyuan Fu,BUPT,https://scs.bupt.edu.cn/info/1097/1781.htm,NOSCHOLARPAGE
 Huizhang Luo,Hunan University,http://csee.hnu.edu.cn/people/luohuizhang,NOSCHOLARPAGE
 Huizhu Jia,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=6097,NOSCHOLARPAGE
 Hujun Bao,Zhejiang University,http://www.cad.zju.edu.cn/home/bao,NOSCHOLARPAGE
+Hulin Kuang,Central South University,https://faculty.csu.edu.cn/kuanghulin/zh_CN/index.htm,cMoIf18AAAAJ

```

Comment:
Same situation as above. Please clarify and remove any faculty who are unable to solely supervise PhD students.